src/decision/CMakeLists.txt

```diff
@@ -56,7 +56,9 @@ add_executable(decision_node
         subroutines/qualification/include/LocateGate.h
         subroutines/qualification/src/LocateGate.cpp
         subroutines/qualification/include/LineUpWithGate.h
-        subroutines/qualification/src/LineUpWithGate.cpp)
+        subroutines/qualification/src/LineUpWithGate.cpp
+        subroutines/qualification/include/GoThroughGate.h
+        subroutines/qualification/src/GoThroughGate.cpp)
```

Comment:
I missed this back in #36, but noticed a the same thing in #44, so just copying my comment here:
"""
Our convention (and a pretty standard C++ one), is to have the `src` and `include` folders both at top level, ie

`routine/Gate/include/locatingGate.h` -> `include/routine/Gate/locatingGate.h`
`routine/Gate/src/locatingGate.cpp` -> `src/routine/Gate/locatingGate.cpp`
"""
Obviously this isn't a super set-in-stone thing, but it's a fairly normal convention for c++ projects I think

Reply:
refactored

Reply:
Yeah, now that I've made this change I can see it's WAY cleaner. Good call.